Sources/BraveWallet/Crypto/NFT/NFTView.swift

```diff
@@ -187,7 +218,66 @@ struct NFTView: View {
           EmptyView()
         })
     )
-    .background(Color(UIColor.braveGroupedBackground))
+    .background(
+      WalletPromptView(
+        isPresented: $isShowingNFTDiscoveryAlert,
+        primaryButton: .init(
+          title: Strings.Wallet.nftDiscoveryCalloutEnable,
+          action: { _ in
+            nftStore.enableNFTDiscovery()
+            Preferences.Wallet.shouldShowNFTDiscoveryPermissionCallout.value = false
+            isShowingNFTDiscoveryAlert = false
+          }
+        ),
+        secondaryButton: .init(
```

Comment:
Should this button styling match desktop and the `Don't show again` Aurora Bridge popup without the border?
![Simulator Screenshot - iPhone 14 Pro - 2023-05-08 at 17 06 15](https://user-images.githubusercontent.com/5314553/236935416-f305fdb3-0d4b-4eb2-ad60-6f5b268fe2a5.png) | ![IMG_4729](https://user-images.githubusercontent.com/5314553/236935461-f66971dd-5714-469e-b82a-cb2c1c16a859.PNG)
--|--

<img width="519" alt="Screenshot 2023-05-08 at 5 02 30 PM" src="https://user-images.githubusercontent.com/5314553/236935446-8d02dd2d-2c8c-4511-a055-893c941d366d.png">

Reply:
updated in [085e5d2](https://github.com/brave/brave-ios/pull/7400/commits/085e5d224d086abd463da13cc8b83042fb26dcbd)